rfcs/000-rfc-lightweightRFCProcess.md

```diff
@@ -0,0 +1,87 @@
+# Lightweight RFC Process
+
+**Geode-RFC-000**
```

Comment:
Can we skip the dashes? Again I feel like the dashes effect readability. Perhaps just:
```
Geode RFC: 0
```

Reply:
The dashes are markdown for making it bold. So it's a trade off between making the interpreted version more readable vs. the raw version. I am fine going either way.

Reply:
I don't mean the `**`, I mean the `-` used in the name. So change to this MD:
`**Geode RFC 0**`
should render:
**Geode RFC 0**